packages/react-native/template/metro.config.js

```diff
@@ -1,11 +1,30 @@
-const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
+const {getDefaultConfig} = require('@react-native/metro-config');
 
 /**
  * Metro configuration
  * https://facebook.github.io/metro/docs/configuration
  *
  * @type {import('metro-config').MetroConfig}
  */
-const config = {};
+const config = getDefaultConfig(__dirname);
 
-module.exports = mergeConfig(getDefaultConfig(__dirname), config);
+// When enabled, the optional code below will allow Metro to resolve
+// and bundle source files with TV-specific extensions
+// (e.g., *.ios.tv.tsx, *.android.tv.tsx, *.tv.tsx)
+//
+// Metro will still resolve source files with standard extensions
+// as usual if TV-specific files are not found for a module.
+//
+// This code is not enabled by default, since it will impact bundling performance,
+// but is available for developers who need this capability.
+//
+/*
+const originalSourceExts = config.resolver.sourceExts;
+const tvSourceExts = [
+  ...originalSourceExts.map((e) => `tv.${e}`),
+  ...originalSourceExts,
+];
+config.resolver.sourceExts = tvSourceExts;
```

Comment:
How would this work when bundling for mobile? If I understand this correctly users should enable this when bundling for tv?

Reply:
Correct, this should only be enabled when building for TV. In [the Expo example](https://github.com/expo/examples/blob/master/with-tv/metro.config.js) I enable it with an environment variable. I'll come up with a non-Expo-dependent way to show that here.

Reply:
Done.